rbac/common/proposal/proposal_address.py

```diff
@@ -40,6 +44,11 @@ def relationship_type(self):
         """The related type from AddressSpace implemented by this class"""
         return addresser.RelationshipType.ATTRIBUTES
 
+    @property
+    def _state_container_prefix(self):
+        """Proposal container is plural (ProposalsContainer)"""
+        return self._state_object_name + "s"
```

Comment:
Should this happen in init()? It seems odd to be concatenating an 's' onto another property on its way out. Maybe an additional property?

Fundamentally, the idea is that data should flow through the system without alteration except in obvious places (ie transformers). If you are going to keep this, then the docstring should at least make it clear what it is doing internally.

Reply:
It’s a property that can be overridden.
It’s a hint for the class to know the state data schema implemented by that address type.